ios/RNSScreenStackHeaderConfig.mm

```diff
@@ -254,7 +254,7 @@ - (void)updateHeaderStateInShadowTreeInContextOfNavigationBar:(nullable UINaviga
 - (NSDirectionalEdgeInsets)computeEdgeInsetsOfNavigationBar:(nonnull UINavigationBar *)navigationBar
 {
   NSDirectionalEdgeInsets navBarMargins = [navigationBar directionalLayoutMargins];
-  NSDirectionalEdgeInsets navBarContentMargins = [navigationBar.rnscreens_findContentView directionalLayoutMargins];
+  NSDirectionalEdgeInsets navBarContentMargins = [navigationBar rnscreens_sumTitleControlDirectionalMargins];
```

Comment:
First thing is that if it had worked before iOS 26, I would not change that, just to reduce potential regressions.

Reply:
Just check for the availbility of iOS 26 runtime.